lib/rules/space-in-parens.js

```diff
@@ -115,38 +118,30 @@ module.exports = {
          * @returns {boolean} True if the token is one of the exceptions for the closer paren
          */
         function isCloserException(token) {
-            return token.type === "Punctuator" && exceptions.closers.indexOf(token.value) >= 0;
+            return exceptions.closers.includes(token.value);
         }
 
         /**
-         * Determines if an opener paren should have a missing space after it
-         * @param {Object} left The paren token
-         * @param {Object} right The token after it
-         * @returns {boolean} True if the paren should have a space
+         * Determines if the first token is a Punctuator, returns second token if not
+         * @param {Object} first The first token (which has its type checked)
+         * @param {Object} second The second token
+         * @returns {Object} Token the first token is returned if it is a punctuator, else second
          */
-        function shouldOpenerHaveSpace(left, right) {
-            if (sourceCode.isSpaceBetweenTokens(left, right)) {
-                return false;
+        function checkPunctuator(first, second) {
+            if (first.type !== "Punctuator") {
+                return second;
             }
-
-            if (ALWAYS) {
-                if (astUtils.isClosingParenToken(right)) {
-                    return false;
-                }
-                return !isOpenerException(right);
-            }
-            return isOpenerException(right);
-
+            return first;
         }
 
         /**
-         * Determines if an closer paren should have a missing space after it
-         * @param {Object} left The token before the paren
-         * @param {Object} right The paren token
-         * @returns {boolean} True if the paren should have a space
+         * Determines if an opening paren has a space immediatley after it
+         * @param {Object} left The paren token
+         * @param {Object} right The token after it
+         * @returns {boolean} True if the opening paren is missing a required space
          */
-        function shouldCloserHaveSpace(left, right) {
-            if (astUtils.isOpeningParenToken(left)) {
+        function openerMissingSpace(left, right) {
```

Comment:
While we're in this space, could we improve the parameter names? Names like `openingParenToken` and `tokenAfterOpeningParen` would be much easier to read than `left` and `right`.

Reply:
Yup, makes sense